runtime/litmus/src/lib.rs

```diff
@@ -703,6 +703,7 @@ impl pallet_bridge::Config for Runtime {
 	type Currency = Balances;
 	type ProposalLifetime = ProposalLifetime;
 	type TreasuryAccount = TreasuryAccount;
+	type WeightInfo = ();
```

Comment:
we can now write `weights::pallet_bridge::WeihtInfo<Runtime>` or something like that

Same applies to other runtimes